models/articles.model.js

```diff
@@ -2,7 +2,22 @@ const db = require("../db/connection");
 
 exports.selectArticleWithID = async (article_id) => {
   const articleFromID = await db.query(
-    "SELECT * FROM articles WHERE article_id = $1",
+`
+    SELECT
+      articles.author,
+      articles.title,
+      articles.article_id,
+      articles.topic,
+      articles.body,
+      articles.created_at,
+      articles.votes,
+      articles.article_img_url,
+      COUNT(comments.comment_id) AS comment_count
```

Comment:
Almost perfect, don't forget to use  `CAST(COUNT(comment_id) AS INT)` to make sure this comes out of the query as an integer!